projects/demo/src/modules/components/input-date-time/examples/7/index.ts

```diff
@@ -0,0 +1,38 @@
+import {AsyncPipe, JsonPipe} from '@angular/common';
+import {Component} from '@angular/core';
+import type {AbstractControl, ValidationErrors, ValidatorFn} from '@angular/forms';
+import {FormControl, FormGroup, ReactiveFormsModule} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiDay} from '@taiga-ui/cdk';
+import {TuiError} from '@taiga-ui/core';
+import {TuiFieldErrorPipe} from '@taiga-ui/kit';
+import {TuiInputDateTimeModule} from '@taiga-ui/legacy';
+
+const completeDateTimeValidator: ValidatorFn = (
+    control: AbstractControl,
+): ValidationErrors | null =>
+    control.value.every(Boolean) ? null : {incompleteDateTime: true};
+
+@Component({
+    standalone: true,
+    imports: [
+        AsyncPipe,
+        JsonPipe,
+        ReactiveFormsModule,
+        TuiError,
+        TuiFieldErrorPipe,
+        TuiInputDateTimeModule,
+    ],
+    templateUrl: './index.html',
+    encapsulation,
+    changeDetection,
+})
+export default class Example {
+    protected readonly testForm = new FormGroup({
+        testValue: new FormControl(
+            [new TuiDay(2017, 2, 15), null],
+            completeDateTimeValidator,
+        ),
+    });
```

Comment:
Let's simplify these lines (to focus user on the main points):

```suggestion
    protected readonly control = new FormControl(
            [new TuiDay(2017, 2, 15), null],
            completeDateTimeValidator,
    );
```

\+ use only `[formControl]="control"` inside template

Reply:
Done